Section_II/competition_and_trophies.tex

```diff
@@ -545,8 +628,8 @@
 
 Teams are strongly encouraged to implement the mitecom team communication protocol which is available at \\
 \url{https://github.com/RoboCup-Humanoid-TC/mitecom} 
```

Comment:
I would not say that mitecom is still recomended, may be confusing

Reply:
Unfortunately, the teams did not decide for either one of them in a previous vote, so at the moment both are recommended in parallel.

Reply:
new question in survey

Reply:
We are adding this as a question for the rule voting again